nautobot/extras/api/serializers.py

```diff
@@ -128,8 +133,14 @@ def get_field_names(self, declared_fields, info):
 
     @extend_schema_field(serializers.URLField())
     def get_notes_url(self, instance):
-        notes_url = get_route_for_model(instance, "notes", api=True)
-        return reverse(notes_url, args=[instance.id], request=self.context["request"])
+        try:
+            notes_url = get_route_for_model(instance, "notes", api=True)
+            return reverse(notes_url, args=[instance.id], request=self.context["request"])
+        except NoReverseMatch:
+            logger.warning(
+                "Notes feature is not available for this model. Please implement it before including NotesSerializerMixin"
```

Comment:
Would be good to make this more detailed - include the name of the model in question, and explain what "implement it" entails?